totest-manager.py

```diff
@@ -802,7 +804,9 @@ class ToTestFactoryPowerPC(ToTestBase):
 
     image_products = [ImageProduct('opensuse-tumbleweed-image:lxc', ['ppc64le'])]
 
-    container_products = [ImageProduct('opensuse-tumbleweed-image:docker', ['ppc64le'])]
+    container_products = [ImageProduct('opensuse-tumbleweed-image:docker', ['ppc64le']),
+                          ImageProduct('kubic-kured-image', ['ppc64le']),
```

Comment:
https://build.opensuse.org/package/show/openSUSE:Factory:PowerPC/kubic-kured-image

At least kured-image for PowerPC is in state excluded; so this can't work

Reply:
That's because `:PowerPC/kubic-kured-image` is not a link yet and images has no linkedbuild.

Reply:
good catch; fixed for :ARM and :PowerPC - kured should thus be buiding